hat/assets/js/apps/Iaso/domains/instances/index.js

```diff
@@ -166,7 +166,6 @@ const Instances = ({ params }) => {
                 tableColumns={tableColumns}
             />
 
-            {fetching && <LoadingSpinner />}
```

Comment:
Maybe repove the unused import of `LoadingSpinner `. 
Also `fetching`, `loadingList` and  `fetchingDetail` are not used anymore